optaplanner-benchmark/src/test/resources/org/optaplanner/benchmark/config/testdataBenchmarkConfig.xml

```diff
@@ -1,11 +1,17 @@
 <plannerBenchmark>
-  <benchmarkDirectory>local/data/nqueens</benchmarkDirectory>
+  <benchmarkDirectory>target/benchmarkTest</benchmarkDirectory>
   <warmUpSecondsSpentLimit>0</warmUpSecondsSpentLimit>
   <solverBenchmark>
     <name>Entity Tabu Search</name>
     <solver>
-      <solutionClass>org.optaplanner.examples.nqueens.domain.NQueens</solutionClass>
-      <entityClass>org.optaplanner.examples.nqueens.domain.Queen</entityClass>
+      <solutionClass>org.optaplanner.core.impl.testdata.domain.TestdataSolution</solutionClass>
+      <entityClass>org.optaplanner.core.impl.testdata.domain.TestdataEntity</entityClass>
+      <scoreDirectorFactory>
+        <scoreDrl>org/optaplanner/core/api/solver/testdataScoreRules.drl</scoreDrl>
+      </scoreDirectorFactory>
+      <termination>
+        <secondsSpentLimit>20</secondsSpentLimit>
```

Comment:
Do we use this benchmark config for tests that run the benchmark or just for verifying the configuration can be built? If it's the former we should use more deterministic termination.

Reply:
Actually, the others use a termination of 0 seconds. I am going to set this one to 0 seconds too, even though it doesn't run (it just for comparison of before and after reading).